BluetraceConfig.swift

```diff
@@ -1,26 +1,20 @@
 //
 //  BluetraceConfig.swift
-//  OpenTrace
+//  CovidSafe
+//
+//  Copyright © 2020 Australian Government. All rights reserved.
+//
```

Comment:
That is.. not how the GPL works

Reply:
Depends on if they got a license from Singapore Government though?